tools/Environments/DevHome.Environments/ViewModels/LandingPageViewModel.cs

```diff
@@ -79,13 +86,17 @@ public partial class LandingPageViewModel : ObservableObject, IDisposable
         ComputeSystemsView = new AdvancedCollectionView(ComputeSystems);
     }
 
+    public void Initialize(StackedNotificationsBehavior notificationQueue)
+    {
+        _notificationService.Initialize(notificationQueue);
+
+        // To Do: Need to give the users a way to disable this, if they don't want to use Hyper-V

```

Comment:
Is this TODO still valid?  If it is, please make an issue and include the issue number in this comment.

Reply:
Wasn't added by me, but I think it is still valid. Will make an issue.